base/inc/fields/number.class.php

```diff
@@ -44,10 +60,22 @@ protected function get_input_classes() {
 	}
 
 	protected function sanitize_field_input( $value, $instance ) {
-		if ( ! empty( $value ) ) {
-			return ( float ) ( ! empty( $this->abs ) ? abs( $value ) : $value );
-		} else {
+
+		if ( empty( $value ) ) {
 			return false;
 		}
+
+		if ( ! empty( $this->min ) ) {
+			$value = max( $value, $this->min );
+		}
+
+		if ( ! empty( $this->max ) ) {
+			$value = min( $value, $this->max );
+		}
+
+		if ( ! empty( $this->abs ) ) {
```

Comment:
Maybe we can remove the abs property? It is not documented on https://siteorigin.com/docs/widgets-bundle/form-building/form-fields/ and to ensure the number is positive you can now use min = 0

Reply:
Due to backwards compatibility reasons and expectations, this isn't possible - it has to be retained.